tests/face_test.py

```diff
@@ -0,0 +1,74 @@
+from socket import AF_INET, AF_INET6
+from _pytest.config import main
+import asyncio
```

Comment:
Sort imports with `isort`.
Moreover, you should add autopep8 and isort into CI scripts to prevent future mistakes.